<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5Databases</class>
 <widget class="QWidget" name="CE5Databases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="leHost">
     <property name="Field" stdset="0">
      <string>f_host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="Primary" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit_2">
     <property name="Field" stdset="0">
      <string>f_name</string>
     </property>
     <property name="First" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Schema</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit_3">
     <property name="Field" stdset="0">
      <string>f_description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="C5LineEditWithSelector" name="leSchema">
     <property name="Field" stdset="0">
      <string>f_db</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="C5LineEditWithSelector" name="leUsername">
     <property name="Field" stdset="0">
      <string>f_user</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="C5LineEditWithSelector" name="lePassword">
     <property name="Field" stdset="0">
      <string>f_password</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Main</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="C5CheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
     <property name="Field" stdset="0">
      <string>f_main</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="btnCheckConnection">
     <property name="text">
      <string>Test connection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5CheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">c5checkbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>leHost</tabstop>
  <tabstop>leSchema</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>btnCheckConnection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
